v3/docs/07-advanced/h-ss58/index.mdx

```diff
@@ -7,28 +7,118 @@ category: advanced
+SS58 is designed to be extensible for this reason.
+
+## Format in detail
+
+This page outlines the implementation of [`Ss58Codec` in Substrate](https://paritytech.github.io/substrate/master/sp_core/crypto/trait.Ss58Codec.html).
+Also of note is the cannonical [SS58 registry](https://github.com/paritytech/ss58-registry) of adress type mappings to various networks described below.
+
+### Basic format
+
+The basic format conforms to:
+
+```
+base58encode ( concat ( <address-type>, <address>, <checksum> ) )
+```
+
+That is, the concatenated byte series of address type, address and checksum then passed into a base-58 encoder.
+The `base58encode` function is implemented exactly as [defined](https://en.wikipedia.org/wiki/Base58) in Bitcoin and IPFS, using the same alphabet as both.
+
+### Address type
+
+The `<address-type>` is one or more bytes that describe the precise format of the following bytes.
+
+Currently, there exist several valid values:
+
+- 00000000b..=00111111b (0..=63 inclusive): Simple account/address/network identifier.
+  The byte can be interpreted directly as such an identifier.
+- 01000000b..=01111111b (64..=127 inclusive): Full address/address/network identifier.
+  The lower 6 bits of this byte should be treated as the upper 6 bits of a 14 bit identifier value, with the lower 8 bits defined by the following byte. 
+  This works for all identifiers up to 2**14 (16,383).
+- 10000000b..=11111111b (128..=255 inclusive): Reserved for future address format extensions.
+
+The latter (42) is a "wildcard" address that is meant to be equally valid on all Substrate networks that support fixed-length addresses.
+For production networks, however, a network-specific version may be desirable to help avoid the key-reuse between networks and some of the problems that it can cause.
+Substrate nodes will default to printing keys in address type 42, though alternative Substrate-based node implementations (e.g. Polkadot) may elect to default to some other type.
+
+### Address Formats for Substrate
+
+There are 16 different address formats, identified by the length (**in bytes**) of the total payload (i.e. including the checksum).
+
+| Total | Type | Raw Account | Checksum |
+|:-----:|:----:|:-----------:|:--------:|
+| 3     | 1    | 1           | 1        |
+| 4     | 1    | 2           | 1        |
+| 5     | 1    | 2           | 2        |
+| 6     | 1    | 4           | 1        |
+| 7     | 1    | 4           | 2        |
+| 8     | 1    | 4           | 3        |
+| 9     | 1    | 4           | 4        |
+| 10    | 1    | 8           | 1        |
+| 11    | 1    | 8           | 2        |
+| 12    | 1    | 8           | 3        |
+| 13    | 1    | 8           | 4        |
+| 14    | 1    | 8           | 5        |
+| 15    | 1    | 8           | 6        |
+| 16    | 1    | 8           | 7        |
+| 17    | 1    | 8           | 8        |
+| 34    | 1    | 32          | 2        |
```

Comment:
```suggestion
| 33    | 1    | 32          | 2        |
```
Not sure what is right here, but needs to add up

Reply:
your correction seems right to me